daffodil-test/src/test/scala/org/apache/daffodil/section15/choice_groups/TestChoice.scala

```diff
@@ -38,6 +38,9 @@ class TestChoice {
 
   import TestChoice._
 
+  @Test def test_choice_noBranch (): Unit = { runnerCH.runOneTest("choice_noBranch") }
+
+  @Test def test_optionalChoice00(): Unit = { runnerCH.runOneTest("optionalChoice00") }
```

Comment:
Rick, you need to delete test_optionalChoice00 since the "optionalChoice00" test doesn't exist.  This is the reason why the PR checks are failing with the message

```
[error] Test org.apache.daffodil.section15.choice_groups.TestChoice.test_optionalChoice00 failed: org.apache.daffodil.tdml.TDMLExceptionImpl: test optionalChoice00 was not found., took 0.001 sec
[error]     at org.apache.daffodil.tdml.TDMLException$.apply(TDMLException.scala:31)
[error]     at org.apache.daffodil.tdml.DFDLTestSuite.runOneTest(TDMLRunner.scala:388)
[error]     at org.apache.daffodil.tdml.Runner.runOneTest(RunnerFactory.scala:144)
[error]     at org.apache.daffodil.tdml.Runner.runOneTest(RunnerFactory.scala:150)
[error]     at org.apache.daffodil.section15.choice_groups.TestChoice.test_optionalChoice00(TestChoice.scala:43)
[error]     ...
```

Reply:
Yep, thanks John.